app/code/Magento/Sales/Model/Order/Payment/Transaction.php

```diff
@@ -833,7 +833,11 @@ public function getTransactionId()
     }
 
     /**
-     * {@inheritdoc}
+     * Set Transaction ID
```

Comment:
Please, revert changes related to @inheritdoc. Replace {@inheritdoc} with @inheritdoc